eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -561,7 +558,7 @@ function Invoke-GenerateAndBuildSDK () {
     Write-Host "service:$service, serviceType:$serviceType"
 
     if (!$readmeFile.StartsWith("http") -And !(Test-Path -Path $readmeFile)) {
-        Write-Error "[ERROR] readme file '$readmeFile' does not exist. Please provide a valid readme file path."
+        Write-Host "[ERROR] readme file '$readmeFile' does not exist. Please provide a valid readme file path."
```

Comment:
`$DotNetSupportChannelLink` needs to be updated to point to [Code Generation - .NET](https://teams.microsoft.com/l/channel/19%3Aacbd512e57bd475198ea6bf4564599e3%40thread.skype/Code%20Generation%20-%20.NET?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121).   It should not be pointing to the main .NET channel.   

I don't have rights to see that alias in `aka.ms`.   Not sure who does.   

@benbp, @weshaggard, @ronniegeraghty:  Any ideas?

Reply:
The link sends me here: https://learn.microsoft.com/en-us/entra/identity/authentication/concept-mandatory-multifactor-authentication?tabs=dotnet#client-libraries

I'm guessing the CI is just saying you shouldn't use the `UsernamePasswordCredential` type anymore. Looks like is picking it up in files that aren't changed in this PR.

Reply:
Ahh, I don't have access to it either. 😟

Reply:
![image](https://github.com/user-attachments/assets/94c062de-a29b-4fd2-908c-f338f4f8fa1a)

@raych1 can you update this? Also might want to change `donet` to `dotnet` and add azsdk as a group owner.